app/views/govuk_publishing_components/components/_chat_entry.html.erb

```diff
@@ -28,22 +28,17 @@
 %>
 
 <%= tag.div(**component_helper.all_attributes) do %>
-  <svg class="gem-c-chat-entry__image" viewBox="0 0 104 104" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
-    <rect width="100%" height="100%" fill="#D2E2F1" />
-    <path d="M37.5733 65.7452L51.6131 77.9999H27.9707L37.5733 65.7452Z" fill="#1D70B8" />
-    <path d="M52 78C66.3594 78 78 66.3594 78 52C78 37.6406 66.3594 26 52 26C37.6406 26 26 37.6406 26 52C26 66.3594 37.6406 78 52 78Z" fill="#1D70B8" />
-    <mask id="mask0_48346_4836" style="mask-type: alpha" maskUnits="userSpaceOnUse" x="40" y="58" width="24" height="10">
-      <rect x="41.2962" y="59.1165" width="21.4029" height="6.98153" fill="#D9D9D9" stroke="#F499BE" stroke-width="1.89313" />
+  <svg class="gem-c-chat-entry__image" width="52" height="52" viewBox="0 0 52 52" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
+    <path d="M11.5713 39.7452L25.6112 51.9999H1.96875L11.5713 39.7452Z" fill="#1D70B8"/>
+    <path d="M26 52C40.3594 52 52 40.3594 52 26C52 11.6406 40.3594 0 26 0C11.6406 0 0 11.6406 0 26C0 40.3594 11.6406 52 26 52Z" fill="#1D70B8"/>
+    <mask id="mask0_51852_3730" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="14" y="32" width="24" height="10">
```

Comment:
I've noticed that there seems to be an error cropping up in the console:

`Refused to apply inline style because it violates the following Content Security Policy directive...`

IIRC, I think I had this happen to me too on an earlier PR and removing `style="mask-type:alpha"` seemed to resolve things. I've just tried removing it locally and the error has gone and everything seems to look okay. If everything still looks okay to you after removing, would you be okay with taking it out?

Reply:
Good spot. Updated.